src/man/firejail.1.in

```diff
@@ -1887,11 +1887,24 @@ Disable printers.
 
 .TP
 \fB\-\-noprofile
-Do not use a security profile.
+Do not load any security profile at all.
```

Comment:
vs. "Do load an empty security profile."?

Reply:
> vs. "Do load an empty security profile."?

Not sure what you mean, so I just reverted it.

Also added a note about profile debugging.

Reply:
No profile is equivalent to an empty profile. I'm not sure what is more clear to users.